tests/platform_tests/args/cont_warm_reboot_args.py

```diff
@@ -0,0 +1,19 @@
+def add_cont_warm_reboot_args(parser):
+    '''
+    Adding arguments required for continuous warm-reboot test case
+    '''
+    parser.addoption(
+        "--cont_reboot_limit",
+        action="store",
+        type=int,
+        default=10,
+        help="Number of iterations of warm-reboot",
+    )
+
+    parser.addoption(
+        "--cont_reboot_delay",
```

Comment:
expand cont to continuous per change above.

Reply:
Change added.